examples/remix/app/root.tsx

```diff
@@ -1,4 +1,5 @@
-import type { MetaFunction } from "@vercel/remix";
+import { cssBundleHref } from "@remix-run/css-bundle";
```

Comment:
I tried to build this and got an error:

```
✘ [ERROR] Could not resolve "@remix-run/css-bundle"

    app/root.tsx:1:30:
      1 │ import { cssBundleHref } from "@remix-run/css-bundle";
        ╵                               ~~~~~~~~~~~~~~~~~~~~~~~
```

Is there a future flag we need to enable or something?

Reply:
I forgot to include the dependency 🙈